searx/settings.yml

```diff
@@ -2094,7 +2098,6 @@ engines:
     engine: yacy
     categories: images
     search_type: image
-    base_url: https://yacy.searchlab.eu
```

Comment:
Are we intentonally using only `https://yacy.searchlab.eu` (defined in the python module) for the images category by default?